### PR TITLE
pt-br translation for QuickFacts and Story58

### PR DESCRIPTION
**Pull request by netrik182**
The translation was done based on the English text. Links and highlights were taken from `es/storyboard` folder.

### CODE REVIEW

**Comment by erciccione** [comment]
Mybe @rmbb could review?

**Comment by lh1008** [comment]
Hey @netrik182,

Thank you for the translation. I was taking a look at the Spanish translation and we need to remove the links. The Spanish translation was taken from the first draft of the story. We are now working on a method to submit a final draft before giving it away to translators. Could you please remove the links? 

Thank you :+1: :)

**Comment by netrik182** [comment]
> Hey @netrik182,
> 
> Thank you for the translation. I was taking a look at the Spanish translation and we need to remove the links. The Spanish translation was taken from the first draft of the story. We are now working on a method to submit a final draft before giving it away to translators. Could you please remove the links?
> :)

Yes, no problem @lh1008. What I'm missing though is the so called original English markdown file. It would be good to have it so me and other translators can match the exactly styling. Do you know where I can find it?

**Comment by lh1008** [comment]
@netrik182 yes you are right. The issue is that the markdown I have was used for the Spanish translation and it seemed it was not updated correctly. So I'm looking for the final draft in order to uploaded to the repo :). 

Thank you for the notice.

**Comment by netrik182** [comment]
Thank you @rmbb. I've added your corrections to the PR.

File: monero-outreach-docs/translations/pt-br/storyboard/story58.md [hunk 1]
```diff
@@ -31,20 +31,20 @@ O Monero possui uma leve taxa de inflação, projetada para manter os incentivos
...
 
-Por sua vez, a descentralização é fortalecida em toda a rede. A rede permanece saudável quando se trata da velocidade e da verificação de transações. O cronograma de recompensas adaptativas do Monero, que orienta a taxa de inflação, garante recompensas consistentes e automáticas para que os mineradores não sejam desencorajados por mudanças repentinas em seus lucros. A curva de emissão de novos Moneroj possui um algorítmo que se adapta automaticamente de acordo com as condições de mineração.
+Por sua vez, a descentralização é fortalecida em toda a rede. A rede permanece saudável quando se trata da velocidade e da verificação de transações. O cronograma de recompensas adaptativas do Monero, que orienta a taxa de inflação, garante recompensas consistentes e automáticas para que os mineradores não sejam desencorajados por mudanças repentinas em seus lucros. A curva de emissão de novos Moneroj possui um algoritimo que se adapta automaticamente de acordo com as condições de mineração.
```
Comment (lh1008) [inline monero-outreach-docs/translations/pt-br/storyboard/story58.md]: Hi @netrik182, 

Could you please correct "algoritmo".
:)

File: monero-outreach-docs/translations/pt-br/storyboard/story58.md [hunk 2]
```diff
@@ -31,20 +31,20 @@ O Monero possui uma leve taxa de inflação, projetada para manter os incentivos
...
+Por sua vez, a descentralização é fortalecida em toda a rede. A rede permanece saudável quando se trata da velocidade e da verificação de transações. O cronograma de recompensas adaptativas do Monero, que orienta a taxa de inflação, garante recompensas consistentes e automáticas para que os mineradores não sejam desencorajados por mudanças repentinas em seus lucros. A curva de emissão de novos Moneroj possui um algoritimo que se adapta automaticamente de acordo com as condições de mineração.
 
 ## Todos os mineradores são bem-vindos
 
-Como o equipamento de mineração ASIC dominou o algorítmo de prova de trabalho do Bitcoin, muitos mineradores caseiros foram expulsos. Isso torna a mineração restrita apenas para um grupo seleto de pessoas, com imenso dinheiro e recursos para se beneficiar da mineração de Bitcoin.
+Como o equipamento de mineração ASIC dominou o algoritimo de prova de trabalho do Bitcoin, muitos mineradores caseiros foram expulsos. Isso torna a mineração restrita apenas para um grupo seleto de pessoas, com imenso dinheiro e recursos para se beneficiar da mineração de Bitcoin.
```
Comment (lh1008) [inline monero-outreach-docs/translations/pt-br/storyboard/story58.md]: "algoritmo"

File: monero-outreach-docs/translations/pt-br/storyboard/story58.md [hunk 3]
```diff
@@ -31,20 +31,20 @@ O Monero possui uma leve taxa de inflação, projetada para manter os incentivos
...
 
-Enquanto a mineração ASIC se tornou comum para o Bitcoin, o Monero atualiza frequentemente seu algorítmo de prova de trabalho para desencorajar o uso de equipamentos ASIC. Com essa abordagem, os chips de CPU e GPU permanecem competitivos e lucrativos. Esta foi e continuará sendo a promessa do Monero para a comunidade.
+Enquanto a mineração ASIC se tornou comum para o Bitcoin, o Monero atualiza frequentemente seu algoritimo de prova de trabalho para desencorajar o uso de equipamentos ASIC. Com essa abordagem, os chips de CPU e GPU permanecem competitivos e lucrativos. Esta foi e continuará sendo a promessa do Monero para a comunidade.
```
Comment (lh1008) [inline monero-outreach-docs/translations/pt-br/storyboard/story58.md]: "algoritmo"

File: monero-outreach-docs/translations/pt-br/storyboard/story58.md [hunk 4]
```diff
@@ -31,20 +31,20 @@ O Monero possui uma leve taxa de inflação, projetada para manter os incentivos
...
 
-A última atualização do Monero Core inclui um algorítmo em nível de estado da arte, chamado Bulletproofs. Essa atualização melhora drasticamente a privacidade, a velocidade e os custos de transação do Monero. Todos esses novos benefícios farão do Monero uma moeda ideal à medida que a adoção continua. Características que visam facilitar as transações nunca devem ser subestimadas.
+A última atualização do Monero Core inclui um algoritimo em nível de estado da arte, chamado Bulletproofs. Essa atualização melhora drasticamente a privacidade, a velocidade e os custos de transação do Monero. Todos esses novos benefícios farão do Monero uma moeda ideal à medida que a adoção continua. Características que visam facilitar as transações nunca devem ser subestimadas.
```
Comment (lh1008) [inline monero-outreach-docs/translations/pt-br/storyboard/story58.md]: "algoritmo"

**Comment by erciccione** [comment]
@lh1008 It's not necessary to open another PR since we are talking about the same language. It will be enough to change the name of the final commit

**Comment by lh1008** [comment]
ok thank you @erciccione, @netrik182 please skip my last comment. 

:)

**Comment by netrik182** [comment]
Hey @erciccione is it right, now? I have fixed the typo as @lh1008 spotted and squashed everything into one single commit

**Comment by lh1008** [comment]
Thank you @erciccione. Great work @netrik182.

**Pull request merged**